src/index.js

```diff
@@ -40,3 +53,6 @@ module.exports = class Deviance {
         done();
     }
 };
+
+module.exports.getEnvIndex = getEnvIndex;
+module.exports.getEnvironment = getEnvironment;
```

Comment:
I did a little reading about the best approach when testing what is effectively private functions in modules and we have a number of options:
1. we don't test these function (as they are private)
2. we make them submodules, so they are imported to both this module and they get their own independent tests
3. we make use of something like (rewire)[https://www.npmjs.com/package/rewire] so we can still test without modifying the src code

Can we discuss thoughts, get some feedback or investigate alternatives?

Reply:
Yeah, we shouldn't export methods purely for testing as it changes the public interface of the module. If they are worthy of testing independently then they should be submodules.

Reply:
getEnvIndex now part of getEnvironment. getEnvironment moved to helpers and exported therefore we're still able to test this.